tests/performance/uniq_to_count.xml

```diff
@@ -0,0 +1,10 @@
+<test>
+    <settings><optimize_uniq_to_count>1</optimize_uniq_to_count></settings>
+
+    <query>select uniq(number) from (select DISTINCT number from numbers(100000000))</query>
```

Comment:
You can try to write your queries in this way instead of writing `<settings>` block:
```suggestion
    <query>select uniq(number) from (select DISTINCT number from numbers(100000000)) SETTINGS optimize_uniq_to_count=1</query>
```
I'm not sure it will help but it is worth trying.

Reply:
Thanks I will give a try

Reply:
I just make default `optimize_uniq_to_count` to true.